src/connection/network/proto/concept.rs

```diff
@@ -285,55 +297,50 @@ impl IntoProto<ThingProto> for Thing {
 
 impl TryFromProto<EntityProto> for Entity {
     fn try_from_proto(proto: EntityProto) -> Result<Self> {
-        let EntityProto { iid, entity_type, inferred: _ } = proto;
+        let EntityProto { iid, entity_type, inferred } = proto;
         Ok(Self {
             iid: iid.into(),
             type_: EntityType::from_proto(entity_type.ok_or(ConnectionError::MissingResponseField("entity_type"))?),
+            inferred,
         })
     }
 }
 
 impl IntoProto<EntityProto> for Entity {
     fn into_proto(self) -> EntityProto {
-        EntityProto {
-            iid: self.iid.into(),
-            entity_type: Some(self.type_.into_proto()),
-            inferred: false, // FIXME
-        }
+        EntityProto { iid: self.iid.into(), entity_type: Some(self.type_.into_proto()), inferred: self.inferred }
```

Comment:
Deconstructing here will not let us miss fields in the future:
```suggestion
        let Self { iid, type_, inferred } = self;
        EntityProto { iid: iid.into(), entity_type: Some(type_.into_proto()), inferred }
```